crates/sui-replay/src/lib.rs

```diff
@@ -584,6 +608,26 @@ pub async fn execute_replay_command(
     })
 }
 
+fn get_filepath(tx_digest: TransactionDigest, extension: &str) -> PathBuf {
+    let mut filename = OsString::new();
+    filename.push("replay_output_");
+    filename.push(tx_digest.to_string());
+    filename.push(".");
+    filename.push(extension);
+    let mut output_path = std::path::PathBuf::from(".");
+    output_path.set_file_name(filename);
+    output_path
+}
+
+fn write_to_file(tx_digest: TransactionDigest, content: &String, extension: &str) {
+    let output_path = get_filepath(tx_digest, extension);
+    let mut out_file = File::create(&output_path).expect("Unable to create file");
+    out_file
+        .write_all(content.as_bytes())
+        .expect("Unable to write file");
```

Comment:
Would it be possible to have nicer errors here instead of using `expect` which will panic?

Reply:
For example, if the disk is full would be good to know that, instead of just panicking with unable to write file.